### PR TITLE
quota: implement check subcommand

### PR DESCRIPTION
**Pull request by Hexilee**

### CODE REVIEW

**Review by davies — APPROVED** [review]

File: pkg/meta/base.go [hunk 1]
```diff
@@ -818,7 +819,39 @@ func (m *baseMeta) HandleQuota(ctx Context, cmd uint8, dpath string, quotas map[
...
+			return st
+		}
+		usedInodes := int64(sum.Dirs+sum.Files) - 1
+		usedSpace := int64(sum.Size) - align4K(0) // quota ignore root dir
+		if q.UsedInodes == usedInodes && q.UsedSpace == usedSpace {
+			logger.Infof("quota of %s is consistent", dpath)
+			quotas[dpath] = q
+			return nil
+		}
+		logger.Errorf(
```
Comment (SandyXSD) [inline pkg/meta/base.go:841]: `warnf` is enough

File: pkg/meta/base.go [hunk 2]
```diff
@@ -818,7 +819,39 @@ func (m *baseMeta) HandleQuota(ctx Context, cmd uint8, dpath string, quotas map[
...
+			"%s: quota(%s, %s) != summary(%s, %s)", dpath,
+			humanize.Comma(q.UsedInodes), humanize.IBytes(uint64(q.UsedSpace)),
+			humanize.Comma(usedInodes), humanize.IBytes(uint64(usedSpace)),
+		)
+		if repair {
+			q.UsedInodes = usedInodes
+			q.UsedSpace = usedSpace
+			quotas[dpath] = q
+			logger.Info("reparing...")
+			return m.en.doSetQuota(ctx, inode, q, false)
```
Comment (SandyXSD) [inline pkg/meta/base.go:851]: `doSetQuota` will not update the used stats

**Review by SandyXSD — CHANGES_REQUESTED** [review]

File: pkg/meta/base.go (hunk 1 above)
Comment (Hexilee) [reply pkg/meta/base.go]: replaced

File: pkg/meta/base.go (hunk 2 above)
Comment (Hexilee) [reply pkg/meta/base.go]: It seems the used stats will be updated if the `create` flag is true.

Comment (Hexilee) [reply pkg/meta/base.go]: done

**Review by SandyXSD — APPROVED** [review]

**Pull request merged**